server/db/round.ts

```diff
@@ -163,20 +164,31 @@ const playCard = async (redis: Redis, gameId: string, playerId: string, cardSuit
     // if this was the last card in the trick, evaluate the trick
     const trickComplete = Boolean(trickCards.length + 1 === players.length);
     let trickWinner = null;
-    const roundComplete = trickComplete && trickNumber === roundNumber;
     if (trickComplete) {
         trickWinner = await evaluateTrick(redis, gameId, roundNumber, trickNumber);
-
-        if (!roundComplete) {
```

Comment:
why did you remove all of this?

Reply:
because this used to happen when the last card was played, but now it happens when every player has said ready